src/main/java/com/gregtechceu/gtceu/api/pattern/MultiblockState.java

```diff
@@ -118,6 +191,13 @@ public BlockState getBlockState() {
         return this.blockState;
     }
 
+    /**
+     * Returns the {@link BlockEntity} at the active position.
+     * If not initialized, tries to query it by checking at the in-world coords.
+     * Can return null if the block isn't a BlockEntity.
+     * 
+     * @return tileEntity
+     */
```

Comment:
The internal details _do not matter_ - again this is lazy caching. Arguably don't even need to mention "Can return null", as the nullability annotation already suggests that.
```suggestion
    /**
     * {@return the BlockEntity at the active position}
     * {@code null} if the BlockState is not a BlockEntity
     */
```